prod-devel-rcar.yaml

```diff
@@ -602,3 +607,18 @@ parameters:
             *DDK_SOURCE_OVERRIDES
           domu:
             *DDK_SOURCE_OVERRIDES
+  ANDROID_PREBUILT_DDK:
+    desc: "Use pre-built GPU drivers for Android"
+    "no":
+      default: true
+    "yes":
```

Comment:
Just a suggestion for consideration by the author.

To initialize variables to empty strings in lines 17-19, and then set their values only under "no"/"yes" blocks.
With such approach we will see all settings in one place:
Like this (I DID NOT check correctness of values! Just example):
```
  XT_DOMA_DDK_KM_PREBUILT_MODULE: ""
  XT_DOMA_KERNEL_EXTRA_MODULES: ""
  XT_DOMA_SOURCE_GROUP: ""

...

  ANDROID_PREBUILT_DDK:
    desc: "Use pre-built GPU drivers for Android"
    "no":
      default: true
        variables:
          XT_DOMA_DDK_KM_PREBUILT_MODULE: "../android_kernel/out/android12-5.4/staging/vendor/lib/modules/pvrsrvkm.ko"
          XT_DOMA_KERNEL_EXTRA_MODULES: "imagination"
          XT_DOMA_SOURCE_GROUP: "all"
    "yes":
      overrides:
        variables:
          XT_DOMA_DDK_KM_PREBUILT_MODULE: "../eva/pvr-km/pvrsrvkm.ko"
          XT_DOMA_KERNEL_EXTRA_MODULES: ""
          XT_DOMA_SOURCE_GROUP: "default"
        components:
          doma:
            builder:
              env:
                - "DDK_UM_PREBUILDS=../eva/pvr-um"
```

Reply:
Once again - this is just a proposal for consideration.
I'm OK with implemented approach.

Reply:
Good idea. Added implementation as proposed.